django_forest/resources/utils/csv.py

```diff
@@ -16,13 +16,12 @@ def fill_csv_relationships(self, res, record, data, params):
         for name, value in record['relationships'].items():
             related_res = self.get_related_res(data, value)
             if related_res:
-                res[name] = related_res['attributes'][params[f'fields[{name}]']]
+                res[name] = related_res['id']
```

Comment:
The `related_res` object has a structure like:
```
 {
    ‘type’: ‘example_table_name’, 
    ‘attributes’: {‘example_col_name’: ‘example_value’}, 
    ‘links’: {‘self’: ‘/forest/example_table_name/example_value’}, 
    ‘id’: ‘example_value’
}
```
so it seems simpler to to me get the "example_value" by doing related_res['id']. 

Also, the current implementation is sometimes causing an error because `[params[f'fields[{name}]']` always evaluates to "id", even when the table doesn't have a column named "id".